docs/ref/models/indexes.txt

```diff
@@ -57,6 +57,29 @@ The name of the index. If ``name`` isn't provided Django will auto-generate a
 name. For compatibility with different databases, index names cannot be longer
 than 30 characters and shouldn't start with a number (0-9) or underscore (_).
 
+``db_tablespace``
+-----------------
+
+.. attribute:: Index.db_tablespace
+
+.. versionadded:: 2.0
+
+The name of the :doc:`database tablespace </topics/db/tablespaces>` to use for
+this index. For single field indexes if ``db_tablespace`` isn't provided the
+index is created in the :attr:`~django.db.models.Field.db_tablespace` from a
+``Field`` instance.
+
+.. note::
+    You can use the :setting:`DEFAULT_INDEX_TABLESPACE` setting to specify
+    a default value for ``Field``
+    :attr:`~django.db.models.Field.db_tablespace`.
+
+If :attr:`~django.db.models.Field.db_tablespace` isn't specified in a ``Field``
```

Comment:
If Field.db_tablespace isn't specified (or if the index uses multiple fields), the index is created in tablespace specified in the model's Meta.db_tablespace. If neither of those tablespaces are set, the index is created in the same tablespace as the table.